### ISSUE TITLE
[Suggestion]: an even cooler version of the phasometer

### ISSUE THREAD

**Issue by Pandaboi25x** [nonamecrackers2/crackers-wither-storm-mod#1595]
### Description of feature request

with 2 chorus fruits and a upgraded phasometer you could craft a even more op phasometer that shows how long the ws is distracted for, is chasing and for how long its playing dead in seconds or minutes its chasing, distracted or playing dead

idk why this idea came to my head but i think its cool smh *my grammar escaped while typing this :why:*

**Comment by nonamecrackers2** [comment]
I don't think we need another upgraded variant, instead these could probably be added to the existing upgraded version instead

**Comment by Pandaboi25x** [comment]
> I don't think we need another upgraded variant, instead these could probably be added to the existing upgraded version instead

yeah that sounds good

**Comment by BluePineappleGuy** [comment]
for when its playing dead, it should say "Slain" instead as to not spoil the surprise of stage 6 for newer players.

**Comment by RepositoryLover** [comment]
It should also show the amount of consumed entities.
